### ISSUE TITLE
Function Open with relative path displays -  Uncaught Error: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by AndrewASel** [mbolotov/k6-intellij-plugin#19]
OS: Ubuntu 24.04
IDE: Webstorm 2024.2.4

Use function open with relative path
https://grafana.com/docs/k6/latest/javascript-api/init-context/open/

For example
`const Binary = open("../../DataFiles/Binary.jpg", "b");`

Run test with debug it stop with error

```
Waiting for the debugger to disconnect...
node:fs:580
  return binding.open(
                 ^

Error: ENOENT: no such file or directory, open '../../DataFiles/Binary.jpg'
    at Object.openSync (node:fs:580:18)
    at Object.readFileSync (node:fs:459:35)
    at p (file:///***/.local/share/JetBrains/WebStorm2024.2/k6/bundle/a.js:1:683)
    at file:///*******:7:19
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:323:24) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '../../DataFiles/Binary.jpg'
}
```
Workaround: add full path instead relative, but it require to update a lot of tests and  full path is different on different PCs

**Comment by mbolotov** [comment]
Hi @AndrewASel,
could you provide your project structure?
where are you starting your script from?
the `open` function work good with relative paths if they are resolved correctly.

**Comment by AndrewASel** [comment]
Hello @mbolotov 
This is absolute paths for structure

File path - /home/ubuntu/Documents/K6/DataFiles/Binary.jpg
Test path- /home/ubuntu/Documents/K6/API/Tests/Section/Test.js

In test I use this code:
`const Binary = open("../../../DataFiles/Binary.jpg", "b");`

It works fine in K6, but stop with error when I run test with debug.
BTW there are no errors for import{} from same relative path, error only for Open() function

**Comment by mbolotov** [comment]
does it work in K6 when you run it via the plugin? or you mean it works when you run k6 from the command line?
could you share the working directory as it shown in the run configuration?
![image](https://github.com/user-attachments/assets/05d43a33-d504-4694-beba-170bd6f7bf1d)

**Comment by AndrewASel** [comment]
Yes, it works when I run it from plugin
![image](https://github.com/user-attachments/assets/a8f8aa37-16cd-43a7-ac68-5c594c1997f0)

> could you share the working directory

I checked, it was empty, it works now with this path - /home/ubuntu/Documents/K6/API/Tests/Section
This is if I use Open() only directly in test.

But looks like I have complex option
I import another JS in my test, like
`import { Common } from "../../Common/Common.js";`
and inside this Common I also initialize open() function, this also throw path errors
Sorry for confusion, modified a lot of rows to update with absolute path.

**Comment by mbolotov** [comment]
Hi @AndrewASel,
Could you try this bundle with the fix? 
[k6-1.2.3.zip](https://github.com/user-attachments/files/17587671/k6-1.2.3.zip)
Please update `k6-debug-api` in your project to `1.4.0` version as well.
Previously, the open() function under debugger resolved a relative path against the current working dir. 
Now, it does this against the caller module dir (the same as k6 does).

Normally, you won't need to change the working directory in the run configuration. It should work okay with the auto filled value.

**Comment by AndrewASel** [comment]
Hello @mbolotov 
I run test with debug, plugin version 1.2.3 and k6-debug-api version 1.4.0
Confirm that this issue has been fixed

**Comment by mbolotov** [comment]
fixed in `1.3.0`
